### ISSUE TITLE
MVP - Basic accessibility 

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#32]
### Summary

Ensure MVP meets basic accessibility requirements

### Deadline 

20th Feb

### What needs to be done?

- [ ] Meets Autism accesibility guidelines
- [ ] Meets basic checklist: 

https://www.gov.uk/government/publications/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one/doing-a-basic-accessibility-check-if-you-cant-do-a-detailed-one

gov.uk has a good one 

### Detail

This is necessary for the MVP - but the fully-fledged plattform should have much higher standards of accesibility. 
It is both a high priority of the autistic/turing project community (see [community-recommendations](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/community-recommendations)) and necessary to meet the value of **diversity and inclusion** 

Another useful accesibility checklist:

https://www.deque.com/ 
See also jiraslamas work on this: https://c6q95y.axshare.com/#g=1&p=7-accessibility

**Comment by jhlink** [comment]
Accessibility can be evaluated after the basic wireframe has been completed. 
For now, removing the `Milestone 2` tag.

**Comment by GeorgiaHCA** [comment]
This issue is the final one to chcekoff once all the Milstonee 3 accessibiity features have been checkde for the MVP.

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
